### ISSUE TITLE
Dragging a tab off in space shows the red circle (not allowed) icon

### ISSUE THREAD

**Issue by elsaco** [microsoft/terminal#15438]
### Windows Terminal version

1.18.230522001-preview

### Windows build number

10.0.19045.2965

### Steps to reproduce

Drag a tab.

### Actual Behavior

The not allowed icon (white square and red circle with red diagonal)  is displayed even though the action is successful.

Also, the `Settings` tab can be dragged off resulting in multiple open settings windows.

**Comment by zadjii-msft** [comment]
Yep. It'll sure do that. It's a bug in the platform currently. MSFT:44327911.

This is known in #14900, but I'll explicitly leave this open and tag it as part of that.
